file.go

```diff
@@ -130,6 +130,10 @@ type GoFile struct {
 	initPackagesOnce  sync.Once
 	initPackagesError error
 
+	runtimeText  uint64
+	pclntabAddr  uint64
+	pclntabBytes []byte
+
```

Comment:
I think it's much better to add a `sync.Once` to prevent the race condition. Also, the error we met should also be cached.